packages/tools/pie-wrappers-react/scripts/react-remove-wrapper.js

```diff
@@ -0,0 +1,21 @@
+import { readFile, writeFile } from 'fs';
+import { components } from './react-gen-wrapper.js';
+
+// removes react wrapper from index.ts after dist has built
+function removeReactWrapper (component) {
+    readFile(`../../components/${component.tagName}/src/index.ts`, { encoding: 'utf8', flag: 'r' }, (err, data) => {
+        if (err) throw (err);
+
+        if (data.includes('import * as React')) {
+            const fileData = data.split('import * as React')[0];
+
+            writeFile(`../../components/${component.tagName}/src/index.ts`, fileData, () => {
+                if (err) throw (err);
+            });
+        }
+    });
+}
+
+if (components.length > 0) {
+    components.forEach((component) => removeReactWrapper(component));
+}
```

Comment:
Wouldn't be part of the clean-up process to remove the generated `custom-elements.json`?

Reply:
we could remove it - I wasn't sure whether it was necessary since the old one is removed whenever the new one gets generated, so I've just added it to `gitignore` instead. Do you think it's better completely removed? happy to change 👍

Reply:
^ update on this, I tried to remove the generated custom-elements.json but it doesn't actually remove and replace the file, just repopulates the content inside - my mistake! For this reason though, I think it needs to simply be put inside .gitignore and remain constant 😄

Reply:
turns out I need to push up the custom-elements.json for the CI build to pass since the components repopulate it in the build process 🤦 so I'm removed it for gitignore too!